src/main/java/org/terracotta/statistics/AbstractOperationStatistic.java

```diff
@@ -26,8 +26,9 @@
 import java.util.Set;
 
 /**
+ * {@link #begin()} and {@link #end(Enum)} must be called from the same thread so that latency can be computed.
+ *
  * @author cdennis
- * @implNote {@link #begin()} and {@link #end(Enum)} must be called from the same thread so that latency can be computed.
```

Comment:
Why not an `implNote` anymore?

Reply:
because project failed to build because of the javadoc plugin... :-(

Reply:
It's a bit weird but you are right. You need to add a `maven-javadoc-plugin` configuration.

```
					<tags>
						<tag>
							<name>apiNote</name>
							<placement>a</placement>
							<head>API Note:</head>
						</tag>
						<tag>
							<name>implSpec</name>
							<placement>a</placement>
							<head>Implementation Requirements:</head>
						</tag>
						<tag>
							<name>implNote</name>
							<placement>a</placement>
							<head>Implementation Note:</head>
						</tag>
					</tags>
```